src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -3355,6 +3365,15 @@ private static bool TryGetNumberValue(string strNum, bool hex, bool real, Number
                                 return true;
                             }
 
+                            result = null;
+                            return false;
+                        case NumberSuffixFlags.SignedByte:
+                            if (Math.Abs(multiplier) == 1L && sbyte.TryParse(strNum, style, NumberFormatInfo.InvariantInfo, out sbyte sb))
```

Comment:
Below we have such code - please add the comment there too.

Why we use `1L`? I think we should use "universal" const `1` without suffix. Below too.

Reply:
Different comment there, but yes.

Wasn't sure if c# would care about type of numeric value when testing for equality, actually. :)